extensions/resource-deployment/src/interfaces.ts

```diff
@@ -3,7 +3,8 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 import * as azdata from 'azdata';
-import { SemVer } from 'semver';
+//import { SemVer } from 'semver';
```

Comment:
why are you removing this?

Reply:
not needed in this file any more as we no ITool no longer has a public property for version but it has a property for fullVersion which is a displayable string object.